test/scripts/test-ovnkube-trace.sh

```diff
@@ -0,0 +1,448 @@
+  local replicas="1"
+
+  file=$(mktemp)
+  cat <<EOF >| $file
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${namespace}
+EOF
+  run_kubectl apply -f ${file}
+
+  file=$(mktemp)
+cat <<EOF >| $file
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ${name}
+  namespace: ${namespace}
+  labels:
+    app: ${name}
+spec:
+  replicas: ${replicas}
+  selector:
+    matchLabels:
+      app: ${name}
+  template:
+    metadata:
+      labels:
+        app: ${name}
+    spec:
+      containers:
+      - name: ${name}
+        image: ${CONTAINER_IMAGE}
+        command:
+        - sleep
+        - "3600"
+EOF
+  run_kubectl apply -f ${file}
+  kubectl_wait_deployment ${namespace} ${name}
+
+file=$(mktemp)
+cat <<EOF >| $file
+apiVersion: v1
+kind: Service
+metadata:
+  name: ${name}
+  namespace: ${namespace}
+spec:
+  type: ClusterIP
+  selector:
+    app: ${name}
+  ports:
+    - port: 80
+      targetPort: 80
+EOF
+  run_kubectl apply -f ${file}
+}
+
+# Spawn a test service named 'service-test' with a test deployment named 'service-test'
+# with 1 replica in namespace named 'ovn-kubetrace-service-test'
+spawn_egressip() {
+  local namespace="egressip-kubetrace-pod-test"
+  local name="egressip-pod-test"
+  local replicas="2"
+  local egressip="172.18.0.10"
+
+  local worker_nodes=$(oc get nodes -o name | grep worker)
+  for n in ${worker_nodes}; do
+    oc label $n k8s.ovn.org/egress-assignable="" --overwrite
+  done
+
+  file=$(mktemp)
+  cat <<EOF >| $file
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${namespace}
+  labels:
+    env: ${namespace}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: ${name}
+  name: ${name}
+  namespace: ${namespace}
+spec:
+  replicas: ${replicas}
+  selector:
+    matchLabels:
+      app: ${name}
+  template:
+    metadata:
+      labels:
+        app: ${name}
+    spec:
+      nodeSelector:
+        k8s.ovn.org/egress-assignable: ""
+      containers:
+      - command:
+        - "sleep"
+        - "3600"
+        image: ${CONTAINER_IMAGE}
+        name: ${name}
+EOF
+  run_kubectl apply -f ${file}
+  kubectl_wait_deployment ${namespace} ${name}
+
+  file=$(mktemp)
+  cat <<EOF >| ${file}
+apiVersion: k8s.ovn.org/v1
+kind: EgressIP
+metadata:
+  name: ${name}
+spec:
+  egressIPs: [ "${egressip}" ]
+  namespaceSelector:
+    matchLabels:
+      env: ${namespace}
+EOF
+  run_kubectl apply -f ${file}
+}
+
+# Clean up all namespaces that we created and remove egress assignable labels
+cleanup() {
+  echo "Cleaning up namespaces"
+  kubectl delete ns ovn-kubetrace-pod-test             2>/dev/null || true
+  kubectl delete ns ovn-kubetrace-pod-test-hostnetwork 2>/dev/null || true
+  kubectl delete ns ovn-kubetrace-service-test         2>/dev/null || true
+  kubectl delete ns egressip-kubetrace-pod-test        2>/dev/null || true
+  kubectl delete egressip egressip-pod-test            2>/dev/null || true
+  local worker_nodes=$(oc get nodes -o name | grep worker)
```

Comment:
Please change this to kubectl to ensure it work on kind clusters

Reply:
Thanks, done!